src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -2895,8 +2895,8 @@ subroutine InitHistory(this, bounds, carbon_type)
        do k = i_litr_min, i_litr_max
           this%dwt_frootc_to_litr_c_col(begc:endc,:,k) = spval
           data2dptr => this%dwt_frootc_to_litr_c_col(begc:endc,:,k)
-          fieldname = 'DWT_FROOTC_TO_LITR_'//trim(decomp_cascade_con%decomp_pool_name_short(k))//'_C'
-          longname =  'fine root to '//trim(decomp_cascade_con%decomp_pool_name_long(k))//' litter due to landcover change'
+          fieldname = 'DWT_FROOTC_TO_'//trim(decomp_cascade_con%decomp_pool_name_history(k))//'_C'
+          longname =  'fine root to '//trim(decomp_cascade_con%decomp_pool_name_long(k))//' due to landcover change'
```

Comment:
Here and elsewhere I have now removed the hardwired strings "LITR_" and "litter" and placed them in `decomp_pool_name_history` and `decomp_pool_name_long`.